### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Found by [typos](https://github.com/crate-ci/typos)

### CODE REVIEW

**Comment by jaimergp** [comment]
TIL about `typos`!

File: tests/test_parse_munged_run_export.py [hunk 1]
```diff
@@ -101,7 +101,7 @@
     - 0001-cmake-Install-python-wrapper-exe-for-scripts-on-Wind.patch
     - 0002-cmake-Don-t-generate-.pyc-and-.pyo-files.patch
     - 0003-grc-Remove-global_blocks_path-preference-and-use-pre.patch
-    - 0004-filter-python-Drop-unused-args-argument-overriden-by.patch
+    - 0004-filter-python-Drop-unused-args-argument-overridden-by.patch
```
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply tests/test_parse_munged_run_export.py:104]: Yeah, I was wondering that too.
My feeling was that it doesn't matter here, but maybe @beckermr has opinions on that.

[the remark this answers is not in the document]
Comment (beckermr) [reply tests/test_parse_munged_run_export.py:104]: Yes do not change this one. If we need to, we can add it to the codespell excludes.

**Comment by beckermr** [comment]
We've been using codespell. I guess it missed these.

File: tests/test_parse_munged_run_export.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply tests/test_parse_munged_run_export.py]: I reverted this change.

**Comment by Hofer-Julian** [comment]
> We've been using codespell. I guess it missed these.

Yeah, I opened the PR because I noticed that typos noticed a few more.
On my projects, I've had better experiences with typos than with codespell, so I moved all of them.

**Comment by beckermr** [comment]
Cool. Does typos run in precommit?

**Comment by Hofer-Julian** [comment]
> Cool. Does typos run in precommit?

If you want, you can run it the same way as codespell: https://github.com/crate-ci/typos/blob/master/docs/pre-commit.md

**Comment by beckermr** [comment]
Awesome. I'd be happy to convert over if people want.

**Review by beckermr — APPROVED** [review]

**Pull request merged**